src/likesApi.js

```diff
@@ -0,0 +1,26 @@
+export const postLikes = async (url, mealId) => {
+  const request = await fetch(url, {
+    method: 'POST',
+    credentials: 'same-origin',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: mealId,
+    }),
+  });
+  return request;
+};
+
+const getLikes = async (url) => {
+  const request = await fetch(url);
+  const result = await request.json();
+  return result;
+};
+
+export const likes = (url, span, mealId) => {
+  getLikes(url).then((data) => {
+    data = data.filter((item) => item.item_id === mealId);
+    span.innerHTML = `${data[0].likes} likes`;
+  });
+};
```

Comment:
- [ ]  To the Like button, kindly add `cursor:pointer` to make it cursor
- [ ] The like button doesn't work, it doesn't count the likes as well as display them on the screen
![rr1](https://user-images.githubusercontent.com/55635977/134408575-ad4ce16b-0be7-44c7-82eb-0a34ee71ff66.png)

Reply:
Well solved